src/components/address.container.jsx

```diff
@@ -43,10 +43,13 @@ const AddressContainer = (props) => {
           {address.locality}
           ,&nbsp;
         </span>
-        <span className="address-region">
-          {address.region}
-          ,&nbsp;
-        </span>
+        {(address.region)
```

Comment:
Thanks for the fix! Was it intentional to remove the surrounding span tags?

Reply:
I did not think about that, fix pushed!